extensions/wikia/WikiaHomePage/WikiaHomePageController.class.php

```diff
@@ -479,33 +479,23 @@ public function getMarketingImage( $imgName ) {
 		return $imageUrl;
 	}
 
-	protected function getHubSliderData($params) {
-		$sliderParams = [
-			'module' => WikiaHubsModuleSliderService::MODULE_ID
-		];
-
-		$sliderParams = array_merge( $sliderParams, $params );
-		return $this->app->sendRequest(
-			'WikiaHubsApi',
-			'getModuleData',
-			$sliderParams
-		)->getData();
+	/**
+	 * @return array
+	 * @deprecated
+	 */
+	protected function getHubSliderData() {
+		return [];
 	}
 
 	/**
 	 * Get list of hubs v3 in selected language
 	 *
 	 * @param $langCode
-	 * @return null
+	 * @return array
+	 * @deprecated
 	 */
 	private function getHubsV3List( $langCode ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$langCode". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)